azure-pipelines.yml

```
@@ -17,10 +17,10 @@ jobs:
       if [[ $BUILD_REASON == "Schedule" ]]; then git config --global user.email "types@microsoft.com" && git config --global user.name "TypeScript Bot" && npm run update-codeowners; fi
       git checkout -- .
 
-      npm run test
+      npm run test-all
 
 
-    displayName: 'npm run test'
+    displayName: 'npm run test-all
```

May need close quote?
```suggestion
    displayName: 'npm run test-all'
```

---

heh, thanks =)

I should probably install yaml-mode to get syntax highlighting.